### PR TITLE
Adding a button for removing the parent email on a student's account

### PR DESCRIPTION
**Pull request by daynew**
As a student, I want to be able to remove the parent email associated with my account.
* Adds "Remove" button/link for setting the `parent_email` to `nil`.
* Adds `aria-label`'s and `roles`'s for the "Update" and "Remove" links so they are easier to use with A11y tools.

## Links
- [jira](https://codedotorg.atlassian.net/browse/FND-1249)

## Screenshots
Removing a parent email address from a student's account.
remove_parent_email

## Testing story
* Manually tested on localhost

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by bethanyaconnor** [comment]
It's hard to tell from the gif but does removing the parent email redirect to the homepage? If so, I feel like it should stay on the account page

**Comment by dju90** [comment]
Unfortunately, that may also be the case with updating other account settings fields too - like updating the "School" field redirects to the teacher dashboard. Ideally, we would keep the user on the account settings page.

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -490,6 +490,9 @@ en:
     for_parents_and_guardians: "For Parents and Guardians"
     link_a_parent_email: "Link a parent/guardian email address to this account to stay updated on your child’s progress and projects. You will also be able to use this email address for password recovery."
     parent_guardian_email_label: "Parent/guardian email"
+    remove_parent_email_label: "Remove"
+    remove_parent_email_aria_label: "Remove Parent Email"
```
Comment (Hamms) [inline dashboard/config/locales/en.yml]: I'd suggest that if we feel this level of specificity is necessary for users with screen readers, it's probably also necessary for those without them.

**Review by bethanyaconnor — APPROVED** [review]

File: dashboard/config/locales/en.yml (hunk 1 above)
Comment (daynew) [reply dashboard/config/locales/en.yml]: aria label removed

**Comment by daynew** [comment]
> It's hard to tell from the gif but does removing the parent email redirect to the homepage? If so, I feel like it should stay on the account page

> Unfortunately, that may also be the case with updating other account settings fields too - like updating the "School" field redirects to the teacher dashboard. Ideally, we would keep the user on the account settings page.

I fixed this redirection.

**Pull request merged**